backend/routes/event.routes.ts

```diff
@@ -0,0 +1,32 @@
+import express from "express";
+import {
+  createEvent,
+  createVerifiedEvent,
+  deleteEvent,
+  eventTest,
+  getAllEvents,
+  getAllEventsByOrganization,
+  getAllVerifiedEvents,
+  getEventById,
+  getRecentEvents,
+  updateEvent,
+} from "../controllers/event.controller";
+import { upload } from "../utils/fileUpload";
+
+const router = express.Router();
+
+// app.use(verifyAuthentication); // Use auth middleware for all routes
```

Comment:
i think for now you can just apply the middleware directly to the methods, unless you have a better way. I think its simple and fine for now:

see user.routes.ts for an examples:

```router.post('/logoutStudent', verifyAuthentication, logoutStudent);```